surface/include/pcl/surface/impl/convex_hull.hpp

```diff
@@ -395,7 +395,6 @@ pcl::ConvexHull<PointInT>::performReconstruction3D (
     {
       polygons[dd].vertices.resize (3);
 
-      // Needed by FOREACHvertex_i_
```

Comment:
I think you accidentally removed the comment

Reply:
@kunaltyagi 
Done!